components/browse/ContentCardSection.vue

```diff
@@ -56,7 +56,27 @@
     },
     computed: {
       cards() {
-        return this.section.fields.hasPart.filter(card => card.fields);
+        const cards = this.section.fields.hasPart.filter(card => card.fields);
+        return cards.map(card => {
+          const identifier = card.fields.identifier;
+          const id = card.sys.contentType.sys.id;
+          if (id === 'automatedEntityCard' && identifier.match(/base/) && cards.length === 4) {
```

Comment:
matching /base/ isn't right here. We only want to match agent/person entities.

I reckon you'll want to include entityParamsFromUri from plugins/europeana/entity.js and check that the returned type attribute === `person`. Alternatively copy the regex from that function, but modify it to only match /agent/ URIs to begin with. That might be more efficient, but I'm  not sure if that modified regex should live in this file here or deserves it's own helper function in the entity plugin.

Additionally I don't think this check should be happening in a mapping of cards. As this check of content length === 4 isn't checking that there are actually 4 people cards, it just checks that there are 4 cards, people or otherwise.
I'd suggest adding a new computed property `isPeopleSection` which only returns true/false if all 4 elements of the object returned by cards() are people cards.

Then rather than having additional logic in the browse content card, make the inclusion of the BrowseContentCard conditional based on that boolean. If it's true, instead include the ContentCards component directly here without the BrowseContentCard wrapper. All the browsContentCard does is normalize the fields. 
As we know the people cards will all be of the type automatedEntityCard, just set the ContentCard props from the known required values, like:
```
  <ContentCard
    v-if="isPeopleSection"
    v-for="card in cards"
    :title="card.fields.name"
    ...
    variant="mini"
  />
```
That'll also mean you won't have to do the modification for omitting the description as you just won't be passing it to the card component in the first place.